bundle/manifests/hw-event-proxy-operator.clusterserviceversion.yaml

```diff
@@ -250,16 +249,16 @@ spec:
                 - name: RELEASE_VERSION
                   value: v0.0.1
                 - name: HW_EVENT_PROXY_IMAGE
-                  value: quay.io/redhat-cne/hw-event-proxy:4.10
+                  value: quay.io/openshift/origin-baremetal-hardware-event-proxy:4.10
                 - name: KUBE_RBAC_PROXY_IMAGE
                   value: quay.io/openshift/origin-kube-rbac-proxy:4.10
```

Comment:
Same here. For 4.10 you need to make another PR

Reply:
done for 4.11